### ISSUE TITLE
Diona Nymph Sucks Ass

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#24432]
(WEB REPORT BY: utilitas REMOTE: 206.221.180.138:7777)
# Revision
55c04da5a9a6e99aaca74dabcf0bb5e834778da7

# Description
yes hello I have an issue with the diona nymph I've been sent, it cannot do anything at all except speak and dionae suck to play even when evolved
like jesus fucking christ at least let them use methylin, that you got cunt punted by a backpack full of baton wielding dionae does not mean you have to _DISABLE ALL FUN FOREVER_
# Steps to Reproduce
play nymph
# What you Expected
a fun experience
# What Actually Happened
ass hurt

**Comment by DrSnips** [comment]
Dihoenas mad. (x24)

**Comment by SonixApache** [comment]
\>can ventcrawl
\>can wear a backpack
\>can use stun batons
dionas deserve less, not more

**Comment by utilitas-** [comment]
\>can do the things all other monkeys can
\>except they cannot use methylin and therefore cannot use anything
Fuck off, Sonix.

**Comment by DrSnips** [comment]
It's almost like being in monkey form as a disability. As for the methylin thing that sounds like an actual bug if its not giving the monkey form dex.

**Comment by Dacendeth** [comment]
Meth only works for human monkeys for whatever reason. My guess is that if nymphs could use guns while on meth and you could put one in your backpack that would be very broken. Letting them use meth to use computers/vending machines seems like it would be fine if you could somehow still restrict gun usage.

**Comment by utilitas-** [comment]
If you go out of your way to _get a cooperating nymph_, _make enough of a meth supply for them to live off of_ (while somehow juggling it inside your backpack) and _to arm them_, you deserve to have them. Never, **ever** restrict player's creativity. At most give them additional hoops to jump through, but never remove it outright.

**Comment by DrSnips** [comment]
Given the reputation that nymphs have right now even with the current limitations I'm not sure if I want them to "be creative".

**Comment by utilitas-** [comment]
What reputation? The one of them "not being played at all because they got nerfed to kingdom come"? Wowee, how scary.

**Comment by DrSnips** [comment]
Its already been posted in the 2nd post.

**Comment by SonixApache** [comment]
\>creativity
nymphs have a reputation for being assholes, giving them even more abilities would just make that worse
methylin was a massive mistake

**Comment by utilitas-** [comment]
yes, monkeys should never have been prevented from using anything in the first place, methylin is just arbitrary bullshit.

**Comment by SonixApache** [comment]
monkeys being able to do anything but die is a massive mistake

**Comment by utilitas-** [comment]
admins being able to do anything but whine in OOC is a massive mistake
